Fabric.Authorization.Persistence.SqlServer/Fabric.Authorization.Persistence.SqlServer.csproj

```diff
@@ -33,7 +33,6 @@
     <Folder Include="DependencyInjection\" />
     <Folder Include="Configuration\" />
     <Folder Include="Mappers\" />
-    <Folder Include="Stores\" />
```

Comment:
I wonder why this keeps happening. It used to be the Mappers folder and now its the Stores folder.

Reply:
Yeah it's weird. I'll fix this.